### ISSUE TITLE
Accessibility colors

### ISSUE THREAD

**Issue by russellmilliner** [roomsense/firmware#12]
Dark Blue on black background is not a good color choice for accessibility reasons.  I recommend following some web accessibility guidelines for your color choices.

Maybe allow access to upload/override the default app.css on the device.  This would at least allow us to customize the look/feel of the device UI.
